Services/UserService/UserService.cs

```diff
@@ -0,0 +1,124 @@
+﻿using AutoMapper;
+using webapi.Dtos.Users;
+using webapi.Models;
+
+namespace webapi.Services.UserService
+{
+    /// <summary>
+    /// Implementation of the user service.
+    /// </summary>
+    public class UserService : IUserService
+    {
+        // In-memory storage for users
+        private static List<Users> users = new List<Users>();
+
+        private readonly IMapper _mapper;
+
+        public UserService(IMapper mapper)
+        {
+            _mapper = mapper;
+        }
+
+        /// <summary>
+        /// Retrieves all users.
+        /// </summary>
+        /// <returns>A list of all users.</returns>
+        public async Task<ServiceResponse<List<GetUserDto>>> GetAllUsers()
```

Comment:
change its implementation also according to above comments.

Reply:
Done